package.json

```diff
@@ -50,8 +50,8 @@
   "devDependencies": {
     "@types/cannon": "^0.1.5",
     "browserify": "^14.4.0",
-    "ts-node": "^9.1.1",
+    "ts-node": "^8.10.2",
```

Comment:
Please revert the originial section in the original commit and dont patch old commits with new ones. Its considered bad practise and leads to a confusing git history.